charts/castai-agent/templates/deployment.yaml

```diff
@@ -142,17 +148,6 @@ spec:
             - name: PROVIDER
               value: {{ .Values.provider | quote }}
             {{- end }}
-          {{- range $k, $v := .Values.additionalEnv }}
-            - name: {{ $k }}
-              value: "{{ $v }}"
-          {{- end }}
-          envFrom:
-            - secretRef:
-          {{- if .Values.apiKey }}
-                name: {{ include "castai-agent.fullname" . -}}
-          {{- else }}
-                name: {{ required "apiKey or apiKeySecretRef must be provided" .Values.apiKeySecretRef -}}
-          {{- end }}
```

Comment:
Deleted by accident?

Reply:
No, linter was complaining about duplicate code, this code exists at line: 81 also

Reply:
You're right, it was for a different container within the pod 🤦  re-added this code.